index.html

```diff
@@ -235,5 +244,35 @@ <h2>I'm always interested in hearing about new projects, so if you'd like to cha
       </ul>
       <h3>diegoyon@outlook.com</h3>
     </footer>
+    <div class="desktop-footer">
+      <hr>
+      <ul class="social">
+        <li>
+          <a href="#">
+            <img src="./images/twitter.svg" alt="twitter" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./images/linkedin.svg" alt="linkedin" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./images/medium.svg" alt="medium" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./images/github.svg" alt="github" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./images/angellist.svg" alt="angellist" />
+          </a>
+        </li>
+      </ul>
+    </div>
   </body>
 </html>
```

Comment:
- [ ] Also try to remove the horizontal scroll bar  in order to match The Figma design it's not used for nothing  
![image](https://user-images.githubusercontent.com/61790587/171492054-5a513704-bf31-40d2-a2f9-8e2d1569d6cf.png)